lxc/storage_volume.go

```diff
@@ -1800,6 +1801,7 @@ func (c *cmdStorageVolumeImport) Command() *cobra.Command {
 		`lxc storage volume import default backup0.tar.gz
 		Create a new custom volume using backup0.tar.gz as the source.`))
 	cmd.RunE = c.Run
+	cmd.Flags().StringVarP(&c.flagName, "name", "n", "", i18n.G("Volume name")+"``")
```

Comment:
Instead of the `name` flag here, we should be able to do the same as `lxc import` and allow it as an optional argument.